avaframe/com1DFA/particleTools.py

```diff
@@ -21,20 +21,22 @@
 log = logging.getLogger(__name__)
 
 
-def initialiseParticlesFromFile(cfg, avaDir):
+def initialiseParticlesFromFile(cfg, avaDir, releaseScenario):
     # TODO: this is for development purposes, change or remove in the future
     # If initialisation from file
+
+    print('RELEASE ', releaseScenario)
```

Comment:
Thi seems to be a debut print? If not, convert to log...

Reply:
removed print